rollup.config.js

```diff
@@ -0,0 +1,93 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { getPackages } from '@lerna/project'
+import filterPackages from '@lerna/filter-packages'
+import batchPackages from '@lerna/batch-packages'
+import typescript from '@wessberg/rollup-plugin-ts'
+import commonjs from '@rollup/plugin-commonjs'
+
+import resolve from '@rollup/plugin-node-resolve'
```

Comment:
we have nodejs, json, typescript, etc deps, so we have plugins for all that stuff